code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -311,7 +311,7 @@
 	projectile_type = /obj/item/projectile/bullet/heavy_rifle_408/practice
 
 /obj/item/ammo_casing/heavy_rifle_408/hv
-	desc = "A 8.6mm high-velocity bullet casing."
+	desc = "A 8.6mm high-velocity SLAP bullet casing."
```

Comment:
```suggestion
	desc = "An 8.6mm high-velocity SLAP bullet casing."
```
English is weird. Due to the word eight, should be 'An'. Good opportunity to make sure other 8.6mm casings have the same grammar.